cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1396,9 +1400,16 @@ def test_deprecated_no_advance_component_button(self):
         self.assertNotIn('Advanced', button_names)
 
     @patch('django.conf.settings.DEPRECATED_ADVANCED_COMPONENT_TYPES', ["poll", "survey"])
-    def test_cannot_create_deprecated_problems(self):
+    def test_deprecated_no_advance_component_button(self):
         """
-        Test that we can't create problems if they are deprecated
+        Test that there will be no `Advanced` button on unit page if units are
+        deprecated in settings file provided that they are the only modules in `Advanced Module List`
+        """
+        self._test_deprecated_no_advance_component_button()
+
+    def _test_cannot_create_deprecated_problems(self):
```

Comment:
You can name this `_assert_cannot_create_deprecated_problems()` to follow the naming convention for assert methods like `assertNone` etc.